### PR TITLE
javascript1-week1/meraj

### PR DESCRIPTION
**Pull request by MerajSharifi**
I think it's ready :)

### CODE REVIEW

File: 3_javascript/javascript1/week1/javascript.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+];
+
+function calculateHouseCost(person) {
+    return (
+       person.width * person.height * person.depth * 2.5 * 1000 + person.gardenSizeInM2 * 300
+    )
+};
+
+let peterHouseCost = calculateHouseCost(friendshouse[0]);
```
Comment (elilovna) [inline 3_javascript/javascript1/week1/javascript.js:53]: if the value does not change it is better you use const

File: 3_javascript/javascript1/week1/javascript.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+let juliaHouseCost = calculateHouseCost(friendshouse[1]);
+
+console.log("Peter's house costs:", peterHouseCost);
+console.log("Julia's house costs:", juliaHouseCost);
+
+     //Peter
+if (peterHouseCost > friendshouse[0].price) {
+    console.log("Peter is paying too much.")
```
Comment (elilovna) [inline 3_javascript/javascript1/week1/javascript.js:61]: great that you used template literals 🤩

File: 3_javascript/javascript1/week1/javascript.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
+
+//Age-ify (A future age calculator)
+
+// let yearOfBirth = 1990
+// let yearFuture = 2050
```
Comment (elilovna) [inline 3_javascript/javascript1/week1/javascript.js:5]: This code commented, but I would think this is a solution, right?

File: 3_javascript/javascript1/week1/javascript.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
+
+//Age-ify (A future age calculator)
+
+// let yearOfBirth = 1990
```
Comment (elilovna) [inline 3_javascript/javascript1/week1/javascript.js:4]: Better to use const if value does not chane

File: 3_javascript/javascript1/week1/javascript.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+// let yearFuture = 2050
+// let age = yearFuture - yearOfBirth;
+// console.log(`You will be ${age} years old in ${yearFuture}.`);
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+// let dogYearOfBirth = 2017;
+// let dogYearFuture = 2027;
```
Comment (elilovna) [inline 3_javascript/javascript1/week1/javascript.js:13]: the same comment, better to use const if value does not change

File: 3_javascript/javascript1/week1/javascript.js (hunk 1 above)
Comment (elilovna) [inline 3_javascript/javascript1/week1/javascript.js:53]: better to use `const` here

File: 3_javascript/javascript1/week1/javascript.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+     //Peter
+if (peterHouseCost > friendshouse[0].price) {
+    console.log("Peter is paying too much.")
+} else {
+    console.log(`Peter is paying ${peterHouseCost} which is reasonable`)
+};
+
+    //Julia
+if (juliaHouseCost > friendshouse[1].price) {
+    console.log("Julia is paying too much.")
```
Comment (elilovna) [inline 3_javascript/javascript1/week1/javascript.js:68]: great that you used template literals for `console.log` 👍

**Review by elilovna — CHANGES_REQUESTED** [review]
Fantastic job! 🎉

It's awesome to see that you've mastered the use of arrays, accessing specific values from them, and using template literals for console.log – that's super cool! 😄

For an even cleaner code, I'd recommend switching 'let' to 'const' and uncommenting the code for review. Keep up the great work! 👍

File: 3_javascript/javascript1/week1/javascript.js (hunk 3 above)
Comment (MerajSharifi) [reply 3_javascript/javascript1/week1/javascript.js:5]: yes, I forgot to un-comment them. now is done!

**Pull request closed without merging**